src/Client/Systems/Network/ClientNetwork.cpp

```diff
@@ -19,4 +20,49 @@ namespace Systems {
             }
         }
     }
+
+    void sendPositionRelative(std::size_t /* unused */, std::size_t /* unused */)
+    {
+        constexpr std::size_t delay = 10;
+        static auto clockId         = Registry::getInstance().getClock().create();
+        static std::map<std::size_t, Types::Position> positionsPlayerCached;
+        Registry &registry = Registry::getInstance();
+
+        if (registry.getClock().elapsedMillisecondsSince(clockId) < delay) {
+            return;
+        }
+        auto ids = registry.getEntitiesByComponents({typeid(Types::Position), typeid(Types::Player)});
```

Comment:
```suggestion
        auto ids = registry.getEntitiesByComponents({typeid(Types::Position), typeid(Types::Player)});
        if (ids.empty()) {
            return;
        }
        auto id = ids[0];
```
and don't need for loop